src/Resources/app/storefront/src/js/filter-category-select.plugin.js

```diff
@@ -105,6 +122,120 @@ export default class FilterCategorySelectPlugin extends FilterBasePlugin {
         this.listing.changeListing();
     }
 
+    /**
+     * @param {HTMLObjectElement} iconShell
+     * @param {string} removeClass
+     * @param {string} addClass
+     * @public
+     */
+    toggleCategoryListIconState(iconShell, removeClass, addClass) {
+
+        if (typeof (iconShell) !== undefined && iconShell !== null) {
+            iconShell.classList.remove(removeClass);
+            iconShell.classList.add(addClass);
+        }
+    }
+
+    /**
+     * @param {HTMLObjectElement} checkbox
+     * @public
+     */
+    onChangeCheckBox(checkbox) {
+        const subCategories = checkbox.parentNode.getElementsByClassName('sub-item');
+        const arrow = checkbox.parentNode.querySelector('#arrow');
```

Comment:
Rename to `icon`. Do this for all occurences, as I really don't want to be depending on how something looks in the storefront. Also rename the id of the element from `#arrow` to `#categoryToggleIcon` or something similar.

Reply:
Done